CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ It is based on Tidepool Blip 1.27.
 - YLP-2438 Inconsistency in the daily view between carbs displayed on the graph and Total carbs in the statistics panel
 
 ### Engineering use
+- YLP-2227 Adjustment of the stats on carbs of the dashboard
```

Comment:
This should be in the "Changed" part of the CHANGELOG (it does not exist, you have to create it). 
"Engineering use" is more meant for refactor and things that does not clearly impact the user experience :slightly_smiling_face: